src/messages/formatters.ts

```diff
@@ -79,6 +81,65 @@ export function blockQuote<C extends string>(content: C): `>>> ${C}` {
 	return `>>> ${content}`;
 }
 
+/**
+ * Formats the URL into `<>`, which stops it from embedding.
+ * @param url The URL to wrap.
+ * @returns The formatted content.
+ */
+export function maskedLink<C extends string>(url: C): `<${C}>`;
+
+/**
+ * Formats the URL into `<>`, which stops it from embedding.
+ * @param url The URL to wrap.
+ * @returns The formatted content.
+ */
+export function maskedLink(url: URL): `<${string}>`;
+export function maskedLink(url: string | URL) {
+	return `<${typeof url === 'object' ? url.href : url}>`;
```

Comment:
```suggestion
	return `<${url}>`;
```
[`url.toString()`](https://nodejs.org/dist/latest-v16.x/docs/api/url.html#url_url_tostring)